test/boost/memtable_test.cc

```diff
@@ -814,3 +815,90 @@ SEASTAR_TEST_CASE(failed_flush_prevents_writes) {
 #endif
 }
 
+SEASTAR_TEST_CASE(flushing_rate_is_reduced_if_compaction_doesnt_keep_up) {
+    // The test simulates a situation where N threads issue flushes to N
```

Comment:
this patch should be the last one, after the fix, otherwise it will break git bisect.

Reply:
sorry for the noise, it's actually the last one. github UI confuses me.